### ISSUE TITLE
Add GUI for variable VVV

### ISSUE THREAD

**Issue by helgatheviking** [topdown/VVV-Dashboard#23]
Feature request: I think it would be pretty cool if the dashboard had a gui as a layer for interacting with the Variable VVV site creation script.

**Comment by topdown** [comment]
I agree. There are multiple problems mostly wrapped around permissions with in vagrant.
I have been thinking about it and I am going to try getting as much automated as possible.
But at the end you would still have to do the vagrant commands via command line, since PHP does not have root level permissions to reload the services.
I'm going to try some ideas when I get some time to work on it again.

**Comment by helgatheviking** [comment]
I've been battling vagrant permissions all day, so I can see how that would pose some problems. I always have a lot of trouble getting VVV installed, but elevated command prompt did help. 

Thanks for the dashboard though. :+1:

**Comment by tomjn** [comment]
A note that variable vvv is no longer maintained and doesn't work with VVV 2, the recommended approach is to use site templates by modifying `vvv-custom.yml` then reprovisioning
